### ISSUE TITLE
Arduino ISP Port Issues

### ISSUE THREAD

**Issue by akvhumbu** [wholder/ATTiny10IDE#5]
**Describe the bug**
Hi Wayne, I can't get the ATTiny10IDE to see my Arduino. After selecting my target(ATtiny85), building the sample code(Blink), and setting up the programmer as ArduinoISP, I get the following error when I try to upload the program:      
avrdude: Error: Invalid -P value: '/dev/cu.usbmodem14601'                              
Use -P usb:bus:device
avrdude done.  Thank you.
<img width="576" alt="Screen Shot 2019-05-26 at 11 23 54 AM" src="https://user-images.githubusercontent.com/51036652/58385141-fa2d0b00-7fa8-11e9-9d90-cf166b4fb5f0.png">
<img width="519" alt="Screen Shot 2019-05-26 at 11 24 51 AM" src="https://user-images.githubusercontent.com/51036652/58385142-fbf6ce80-7fa8-11e9-83ac-cf30abe7443a.png">

The Arduinos that Ive tried to use(Uno & Nano) have the Arduino ISP Sketh already uploaded into them and can program the ATtiny85 using the Arduino IDE. I've had no success on both Mac & Windows.

**Comment by wholder** [comment]
Thanks for calling this to my attention.  It looks like a regression due to some refactoring I did some time back.  I've made a change that should fix the issue.  So, please download the latest build and let me know if this solves the prombel for you.

**Comment by akvhumbu** [comment]
Thanks for the quick turn around. I am able to program my ATtiny85. Will a future update have a serial console, and the ability to debug such as that of the DebugWireDebuggerProgrammer?

**Comment by wholder** [comment]
Yes.  That's the plan.  At the moment I'm trying to improve the built in
documentation and smooth out some rough edges in the code.  But, I'm also
thinking through how the interface needs to change to best integrate
debugging.  If you have any suggestions on how to improve the current code
and/or how to best integrate debugging, please let me know.

Wayne

**Comment by akvhumbu** [comment]
Hi Wayne,

I'm certain the IDE has support for libraries aka "#include headerfiles". I have been trying to include libraries other than the AVR & C ones but they are not being linked. Do you know where I should save my header files on the Mac in order for the linker to find them? I've tried saving in the Libraries folder where Arduino IDE looks for, but that doesn't work either.

Regards,

**Comment by wholder** [comment]
No, sorry, the IDE doesn't currently support include files other than for
the built in libraries.  I'm hoping to add this at some point, but I
haven't yet worked out how to do this in a way that will be compatible with
the automatic prototype generation code I've added.

Wayne
